k8s/aat/common/sscs/sscs-track-appeal-redirect-ingress.yaml

```diff
@@ -8,14 +8,14 @@ metadata:
     traefik.ingress.kubernetes.io/redirect-permanent: "true"
     traefik.ingress.kubernetes.io/redirect-regex: ^https://track-appeal.aat.platform.hmcts.net/(.*)
     traefik.ingress.kubernetes.io/redirect-replacement: https://sscs-cor.aat.platform.hmcts.net/validate-surname/:tya/trackyourappeal
-  name: sscs-track-appeal-redirect
+  name: sscs-track-appeal-redirect-ingress-ingress
```

Comment:
is that correct naming covention?

Reply:
it's consistent with the other ingress redirects

Reply:
sorry, just seen double ingress. will fix